lib/reservation_dates.rb

```diff
@@ -0,0 +1,17 @@
+require 'date'
+
+module HotelBookings
+  class Reservation_Dates
```

Comment:
Nitpick: Naming convention says this is named `ReservationDate` for class names-- no underscore, and keep it singular!